src/collection-browser.ts

```diff
@@ -368,24 +372,44 @@ export class CollectionBrowser
     const sortField = SortFieldToMetadataField[this.selectedSort];
     if (!sortField) return;
     this.sortParam = { field: sortField, direction: this.sortDirection };
+
+    this.analyticsHandler?.sendEvent({
+      category: 'collection-browser',
+      action: `selectedSortChanged - sortParam: ${this.sortParam}`,
+    })
   }
 
   private displayModeChanged(
     e: CustomEvent<{ displayMode: CollectionDisplayMode }>
   ) {
     this.displayMode = e.detail.displayMode;
+
+    this.analyticsHandler?.sendEvent({
+      category: 'collection-browser',
+      action: `change viewMode: ${this.displayMode}`
```

Comment:
Could this action be `viewModeChanged: ${this.displayMode}` for consistency with the other events? (Not sure if these actions are existing tags we're already using 😄)